docs/features.md

```diff
@@ -250,7 +264,7 @@ spec:
   authentication:
     "pre-validated-jwt":
       plain:
-        selector: context.metadata_context.filter_metadata.envoy\.filters\.http\.jwt_authn|verified_jwt
+        expression: metadata.filter_metadata.envoy['filters.http.jwt_authn'].verified_jwt
```

Comment:
We've seen issues getting to that data in the wasm-shim when directly requesting that from envoy. 
But would that even work on the authorino side? There is no way the entire filter chain's metadata is sent over... is it? Did that _ever_ worked? 

Also, this should be `metadata.filter_metadata['envoy.filters.http.jwt_authn'].verified_jwt`

Reply:
/cc @adam-cattermole

Reply:
It works in Authorino.

Reply:
> Also, this should be metadata.filter_metadata['envoy.filters.http.jwt_authn'].verified_jwt

Nice catch! Fixing it now. Thanks!